### PR TITLE
Fix tests for `swift-api-digester` integration

### PR DESCRIPTION
**Pull request by neonichu**
Looks like these tests are disabled by default, so some failures snuck in.

### CODE REVIEW

File: Tests/CommandsTests/APIDiffTests.swift [hunk 1]
```diff
@@ -97,7 +97,7 @@ final class APIDiffTests: CommandsTestCase {
             try localFileSystem.writeFileContents(packageRoot.appending(components: "Sources", "Qux", "Qux.swift")) {
                 $0 <<< "public class Qux<T, U> { private let x = 1 }"
             }
-            XCTAssertThrowsCommandExecutionError(try execute(["diagnose-api-breaking-changes", "1.2.3", "-j", "2"], packagePath: packageRoot)) { error in
+            XCTAssertThrowsCommandExecutionError(try execute(["diagnose-api-breaking-changes", "1.2.3"], packagePath: packageRoot)) { error in
```
Comment (neonichu) [inline Tests/CommandsTests/APIDiffTests.swift:100]: Not entirely sure what the `-j 2` was doing there, seems unrelated to the test?

File: Tests/CommandsTests/APIDiffTests.swift [hunk 2]
```diff
@@ -154,7 +154,8 @@ final class APIDiffTests: CommandsTestCase {
             XCTAssertThrowsCommandExecutionError(try execute(["diagnose-api-breaking-changes", "1.2.3"], packagePath: packageRoot)) { error in
                 XCTAssertMatch(error.stdout, .contains("1 breaking change detected in Foo"))
                 XCTAssertMatch(error.stdout, .contains("💔 API breakage: struct Foo has been removed"))
-                XCTAssertMatch(error.stdout, .contains("1 breaking change detected in Bar"))
+                XCTAssertMatch(error.stdout, .contains("2 breaking changes detected in Bar"))
+                XCTAssertMatch(error.stdout, .contains("💔 API breakage: import Baz has been removed"))
```
Comment (neonichu) [inline Tests/CommandsTests/APIDiffTests.swift:158]: This looks like an improvement to the underlying tool

**Comment by neonichu** [comment]
@swift-ci please smoke test

**Review by tomerd — APPROVED** [review]
